content/en/agent/guide/cluster-agent-custom-metrics-server.md

```diff
@@ -0,0 +1,322 @@
+As you create the HPA, the Datadog Cluster Agent parses the manifest and queries Datadog to try to fetch the metric.
+- If there is a typographic issue with your metric name or if the metric does not exist within your Datadog application, the following error is raised:
+	```
+	2018-07-03 13:47:56 UTC | ERROR | (datadogexternal.go:45 in queryDatadogExternal) | Returned series slice empty
+	```
+
+You can run the `datadog-cluster-agent status` command to see the status of the External Metrics Provider process:
+
+```
+  Custom Metrics Provider
+  =======================
+  External Metrics
+  ================
+    ConfigMap name: datadog-hpa
+    Number of external metrics detected: 2
+```
+Note: Errors with the External Metrics Provider process are displayed with this command.
+If you want a more verbose output, run the flare command:
+`datadog-cluster-agent flare`.
+The flare command generates a zip file containing the `custom-metrics-provider.log` where you can see an output as follows:
+
+```
+  Custom Metrics Provider
+  =======================
+  External Metrics
+  ================
+    ConfigMap name: datadog-hpa
+    Number of external metrics detected: 2
+    
+    hpa:
+    - name: nginxext
+    - namespace: default
+    labels:
+    - cluster: eks
+    metricName: redis.key
+    ts: 1532042322
+    valid: false
+    value: 0
+    
+    hpa:
+    - name: nginxext
+    - namespace: default
+    labels:
+    - dcos_version: 1.9.4
+    metricName: docker.mem.limit
+    ts: 1.532042322
+    valid: true
+    value: 268435456
+
+
+```
+ 
+If the metric's flag `Valid` is set to false, the metric is not considered in the HPA pipeline.
+
+- If you see the following mesage when describing the HPA manifest:
+
+```
+Conditions:
+  Type           Status  Reason                   Message
+  ----           ------  ------                   -------
+  AbleToScale    True    SucceededGetScale        the HPA controller was able to get the target's current scale
+  ScalingActive  False   FailedGetExternalMetric  the HPA was unable to compute the replica count: unable to get external metric default/nginx.net.request_per_s/&LabelSelector{MatchLabels:map[string]string{kube_container_name: nginx,},MatchExpressions:[],}: unable to fetch metrics from external metrics API: the server could not find the requested resource (get nginx.net.request_per_s.external.metrics.k8s.io)
+
+```
+
+Then it's likely that you don't have the proper RBAC set for the HPA.
+Make sure that `kubectl api-versions` shows:
+
+```
+autoscaling/v2beta1
+[...]
+external.metrics.k8s.io/v1beta1 
+```
+
+The latter will show up if the Datadog Cluster Agent properly registers as an External Metrics Provider—and if you have the same service name referenced in the APIService for the External Metrics Provider, as well as the one for the Datadog Cluster Agent on port 443.
+Also, make sure you have created the RBAC from the "Register the External Metrics Provider" step.
```

Comment:
You should link to the "Register the External Metrics Provider" step with an anchor.